src/firstvoices/backend/models/characters.py

```diff
@@ -1,9 +1,11 @@
+import g2p
```

Comment:
The g2p package should probably be added to the base requirements file.

Reply:
I can add g2p to the base requirements file but currently there is a Werkzeug dependency conflict between our repo and the most recent versions of g2p. Currently installing g2p via the requirements file with an unspecified version installs g2p version 0.5.20220318

Reply:
Message from Aidan/g2p is that they are sticking with werkzeug 2.0 in order to support python 3.6. So to resolve this, we either need to downgrade or look into an alternative dev server option like gunicorn.